code/datums/emergency_calls/emergency_call.dm

```diff
@@ -21,6 +20,14 @@
 	var/candidate_timer
 	var/cooldown_timer
 	var/spawn_type = /mob/living/carbon/human
+	///The base probability of that ERT spawning, it is changing with monitor state
+	var/base_probability = 0
+	/**How the current_weight change with the monitor state. A big positive number will make the current weight go down drasticly when marines are winning
```

Comment:
Can you start the actual comments on the next line for the sake of formatting please